### PR TITLE
Replace rst in root with md

### LINKED ISSUE #199: Making release description hard due to RestructuredText to Markdown conversion
The [making a release instructions](https://github.com/eWaterCycle/ewatercycle/blob/main/CONTRIBUTING.rst) ask for copying text from the README.rst and CHANGELOG.rst to the GH release description form field. However the GH release form requires Markdown and the files are in RestructuredText.

For now we used pandoc to convert with
```shell
pandoc -s README.rst -o README.md
pandoc -s CHANGELOG.rst -o CHANGELOG.md
```

It would be nice if the conversion would not be needed or at least was documented.

### PR DESCRIPTION
**Pull request by sverhoeven**
Fixes #199

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/).
+Formatted as described on [https://keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
+
+## [Unreleased]
+
+## [1.0.0] (2021-07-21)
```
Comment (Peter9192) [inline CHANGELOG.md:9]: Links don't work

File: README.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+[![Documentation Status](https://readthedocs.org/projects/ewatercycle/badge/?version=latest)](https://ewatercycle.readthedocs.io/en/latest/?badge=latest)
+
+[![PyPI](https://img.shields.io/pypi/v/ewatercycle)](https://pypi.org/project/ewatercycle/)
+
+[![image](https://img.shields.io/badge/fair--software.eu-%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8B-yellow)](https://fair-software.eu)
+
+[![image](https://zenodo.org/badge/DOI/10.5281/zenodo.5119389.svg)](https://doi.org/10.5281/zenodo.5119389)
+
+[![Research Software Directory Badge](https://img.shields.io/badge/rsd-ewatercycle-00a3e3.svg)](https://www.research-software.nl/software/ewatercycle)
```
Comment (Peter9192) [inline README.md]: I would not add newlines between the badges

**Review by Peter9192 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (sverhoeven) [reply CHANGELOG.md:9]: Fixed

File: README.md (hunk 2 above)
Comment (sverhoeven) [reply README.md]: Yeah that looks a bit to spacy

**Review by Peter9192 — APPROVED** [review]

**Pull request merged**